src/html-resource-plugin.js

```diff
@@ -6,23 +6,24 @@ export function getElementName(address) {
 }
 
 export function configure(config) {
-  let viewEngine = config.container.get(ViewEngine);
-  let loader = config.aurelia.loader;
+  const viewEngine = config.container.get(ViewEngine);
+  const loader = config.aurelia.loader;
 
   viewEngine.addResourcePlugin('.html', {
-    'fetch': function(address) {
-      return loader.loadTemplate(address).then(registryEntry => {
-        let bindable = registryEntry.template.getAttribute('bindable');
-        let elementName = getElementName(address);
+    'fetch': function(viewUrl) {
+      return loader.loadTemplate(viewUrl).then(registryEntry => {
+        let bindableNames = registryEntry.template.getAttribute('bindable');
+        const useShadowDOMmode: null | '' | 'open' | 'closed' = registryEntry.template.getAttribute('use-shadow-dom');
+        const name = getElementName(viewUrl);
```

Comment:
Lol, you renamed `bindable` to `bindableNames` while renaming `elementName` to `name`? 😆 

Let's have `elementName`, I think it's general theme of naming convention in Aurelia.

Reply:
I matched the values with params in dynamic element. I would like to keep the names identical, should I update both or only here?

Reply:
I missed that, yeah lets keep it this way. Just small nit anyway 👍

Reply:
ok I won't change the naming and will close this conversation.